src/orders/order-options-response.ts

```diff
@@ -20,6 +21,29 @@ export class OrderOptionsResponse implements IOrderOptionsResponse {
         this.response = response;
     }
 
+    public getAccessories(): IAccessory[] {
+        if (!this.response.accessories || !this.response.accessories.length) {
+            return [];
+        }
+
+        return this.response.accessories.map((accessory) => ({
+            id: accessory.id,
+            articleNumber: accessory.articleNumber,
+            logoUrl: accessory.logoUrl,
+            longDescription: accessory.longDescription,
+            shortDescription: accessory.shortDescription,
+            manufacturer: accessory.manufacturer,
+            model: accessory.model,
+            name: accessory.name,
+            price: accessory.price,
+            assemblyPrice: accessory.assemblyPrice,
+            salePrice: accessory.salePrice,
+            productPageLink: accessory.productPageLink,
+            productPageLinkText: accessory.productPageLinkText,
+            media: accessory.media,
```

Comment:
I think it would be preferable to map the whole `media` model here, using `.map`.

Reply:
Hm, förstår inte riktigt. `media: accessory.media.map((m) => ....),` något sånt tänker du?

Reply:
Hm, förstår inte riktigt. media: accessory.media.map((m) => ....), något sånt tänker du?

Reply:
Yep, exactly!

Reply:
Now I think I get it. This addition makes the code more clear and makes it easy to adjust. I guess that was the point?

Reply:
Fixed in 755ba1d.